usermods/usermod_v2_rotary_encoder_ui_ALT/readme.md

```diff
@@ -50,7 +42,8 @@ To activate this alternative usermod, add `#define USE_ALT_DISPlAY` (NOTE: CASE
 
 ### PlatformIO requirements
 
-Note: the Four Line Display usermod requires the libraries `U8g2` and `Wire`.
+No special requirements.
```

Comment:
Wrong. U8g2 and Wire are needed, regardless of being enabled by default.

Reply:
Is Wire and U8g2 required also for rotary encoder only?

Reply:
Sorry. 4LD requires them, not rotary.